src/Template/Provider/BaseProvider.php

```diff
@@ -179,23 +179,19 @@ protected function requestPackageVersionConstraint(Template\TemplateSource $temp
 
         $this->messenger->writeWithEmoji(
             IO\Emoji::WhiteHeavyCheckMark->value,
-            sprintf(
-                'Well done! You\'ve selected <fg=yellow>%s</>.',
-                $templateSource->getPackage()->getName(),
-            ),
+            sprintf('Well done! You\'ve selected <comment>%s</comment>.', $templateSource->getPackage()->getName()),
         );
 
         $this->messenger->newLine();
-
+        $this->messenger->write(
+            sprintf('Do you require a specific version of <comment>%s</comment>?', $templateSource->getPackage()->getName()),
+        );
         $this->messenger->comment(
-            sprintf(
-                'Do you require a specific version of %s?
-If so, you may specify it here. Leave it empty and we\'ll find a current version for you.',
-                $templateSource->getPackage()->getName(),
-            ),
+            'If so, you may specify it here. Leave it empty and we\'ll find a current version for you.',
         );
-
-        $this->messenger->comment('Example: \'0.2.0\' or \'dev-feature/xyz\'');
+        $this->messenger->newLine();
```

Comment:
I'm not sure I like the new lines here. The whole section seems off at the moment. But let's jut keep it for this version.

![image](https://user-images.githubusercontent.com/2636487/227247909-763e10fb-f84c-4a6d-94e8-90eed822bf1a.png)

Reply:
I agree and just pushed another variant. Let's stick to it for now and run another optimization iteration later.

Reply:
Ok. `cyan` it is for v2. 👍